conans/client/command.py

```diff
@@ -553,7 +556,11 @@ def info(self, *args):
 
             if args.graph:
                 self._outputer.info_graph(args.graph, deps_graph, get_cwd())
-            else:
+            if args.json:
+                json_arg = True if args.json == "1" else args.json
+                self._outputer.json_info(deps_graph, json_arg, get_cwd(), show_paths=args.paths)
```

Comment:
I think we could output always the paths when json output and forbid the `args.paths` and `args.json` together. It simplifies a bit everything and I would expect a json to be always complete.

Reply:
For packages installed as editable access to paths raises. There are two options:
 * keep the argument, if the user asks for paths, it will raise.
 * always True and try/except the paths section, paths won't be returned and the user won't know why.